APIBlueprintGenerator.coffee

```diff
@@ -88,6 +88,47 @@ APIBlueprintGenerator = ->
         description: description,
       }
 
+  # Generate a parameter List for the mustache context from a paw Request
+  #
+  # @param [Request] exchange The paw HTTP request
+  #
+  # @return [Object] The template context object
+  #
+  @parameter = (paw_request) ->
+    parameters = []
+    body = paw_request.body
+    has_body = body.length > 0
+    url_parameters = paw_request.getUrlParameters()
+    for key, value of url_parameters
+      parameters.push({
+        name: key,
+        example: value,
+        type: @isNumber(value)
+        })
+    if has_body
```

Comment:
Thanks for the pull request @AlanLi7991.

In the API Blueprint language, the parameters section is for URI Parameters only. It should *NOT* contain body attributes. These belong in a different section ("Attributes"). This would be a bit of a larger task since you would need to recurse over arrays and objects. It's already tracked by #14.

To do this right, you would need to create an attributes section instead, but I think it would probably be easier to just remove the body attributes becoming parameters and we can tackle that in #14.

The attributes could look something like:

```apib
+ Response 200 (application/json)
    + Attributes (array)
        + (object)
            + name: Kyle
            + company (object)
                + name: Apiary
```

You can find more information on body attributes description in our advanced API Blueprint tutorial: https://apiblueprint.org/documentation/advanced-tutorial.html#attributes